go/oasis-test-runner/scenario/e2e/registry_cli.go

```diff
@@ -442,7 +458,7 @@ func (sc *registryCLIImpl) initNode(childEnv *env.Env, ent *entity.Entity, entDi
 			"--" + cmdRegNode.CfgNodeRuntimeID, testNode.Runtimes[0].ID.String(),
 			"--" + cmdSigner.CfgSigner, fileSigner.SignerName,
 			"--" + cmdSigner.CfgCLISignerDir, entDir,
-			"--" + cmdCommon.CfgDataDir, dataDir,
+			"--" + cmdCommon.CfgConfigFile, cfgFile,
```

Comment:
Why is a config file needed for `registry node init`?

Reply:
It needs the data dir from it and I figured it would be nicer for the users if they didn't have to retype the data dir on the command line, but instead just pass their config file (same for `registry node is-registered`).